### DISCUSSION TITLE
An obscure exception in Flutter app.

### DISCUSSION THREAD

**Discussion by Developer1414** [supabase/supabase#14662] [Questions]
Hi! Why do I keep getting an exception when I call the update method (screenshot 1) (screenshot 2)? I.e. the data in Supabase is updated, but after that an exception is thrown. This happens not only with update, but also with insert and upsert, for example. I don't understand how to fix it.
![Code_yQL08m59iq](https://github.com/supabase/supabase/assets/43788117/8982b2c1-2bb6-4eea-8f21-5ed5ef509089)
![Code_eW66Oui40q](https://github.com/supabase/supabase/assets/43788117/e8491444-295b-4bd0-a4b5-68ea1a8a7767)

**Comment by dshukertjr** [comment]
Hi there,

Could you copy and paste the exact error you are receiving along with StackTrace?
